tests/integ/test_session.py

```diff
@@ -73,6 +73,46 @@ def test_active_session(session):
     assert session == _get_active_session()
 
 
+def test_get_or_create(session):
+    # because there is already a session it should report the same
+    new_session = Session.builder.getOrCreate()
+    assert session == new_session
```

Comment:
add `session.close()`

Reply:
mmm I think that in this case I should not close the session. The getOrCreate is returning the same session object not a new one so in this scenario I need to keep the session.

Reply:
you are right. my bad